responses/_recorder.py

```diff
@@ -143,13 +144,20 @@ def _on_request(
         headers_values = {
             key: value for key, value in requests_response.headers.items()
         }
-        responses_response = Response(
-            method=str(request.method),
-            url=str(requests_response.request.url),
-            status=requests_response.status_code,
-            body=requests_response.text,
-            headers=headers_values,
-        )
+
+        response_params = {
+            "method": str(request.method),
+            "url": str(requests_response.request.url),
+            "status": requests_response.status_code,
+            "body": requests_response.text,
+            "headers": headers_values,
+        }
+
+        # If the header has a content type then pass it in.
+        if content_type := requests_response.headers.get("content-type"):
+            response_params["content_type"] = content_type
```

Comment:
It would be good to have a test covering this change and validating that the Response that is created from the recording has the right parameters/headers.